x-pack/plugins/index_management/server/config.ts

```diff
@@ -39,6 +39,14 @@ const schemaLatest = schema.object(
       schema.never()
     ),
     dev: schema.object({ enableIndexDetailsPage: schema.boolean({ defaultValue: false }) }),
+    enableIndexStats: schema.conditional(
```

Comment:
Depending when https://github.com/elastic/kibana/pull/163786 is merged, I can update as part of this PR or in a follow up PR.